example/server.js

```diff
@@ -2,13 +2,17 @@
 
 var Hapi = require('hapi');
 var Hoek = require('hoek');
+var fs = require('fs');
 var tags = require('../lib/index.js');
-
+var tagsData = JSON.parse(fs.readFileSync('./example/tags.json').toString());
```

Comment:
a yes caching! It's fine for now we can keep the example simple it was just more a note but it might be a good idea to create somewhere an issue with readFile vs require for json file